.github/workflows/sonarqube.yml

```
@@ -12,16 +12,109 @@ jobs:
+          pip install -r backend/requirements.txt
+          pip install pytest pytest-django pytest-cov
+
+      - name: Run migrations
+        working-directory: ./backend
+        run: python manage.py migrate
+
+      - name: Run tests with coverage
+        working-directory: ./backend
+        run: |
+          python -m pytest
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)